File: gaia-sdk-javascript/src/graphql/request/type/BuildInEvaluation.ts


import VariableRegistry from "../../../api/VariableRegistry"
import {Uuid, ISO8601, Struct} from "../../GaiaClient";
import {Order} from "../enumeration/Order";
import {OrderByField} from "../enumeration/OrderByField";
import {EdgeOrderByField} from "../enumeration/EdgeOrderByField";
import {EdgeType} from "../enumeration/EdgeType";

export class BuildInEvaluation extends Array<(_:VariableRegistry) => string> {
public _typeName = "BuildInEvaluation";
    public behaviour = () => { 
        this.push(_ => "behaviour")
    };

    public gaiaquery = () => { 
        this.push(_ => "gaiaquery")
    };

    public render = (registry: VariableRegistry):String => this.map(e => e(registry)).join(" ");
}

File: gaia-sdk-python/graphql/request/type/BuildInEvaluation.py


from typing import Callable
from api.VariableRegistry import VariableRegistry


class BuildInEvaluation(list):

    def behaviour(self):
        self.append(lambda x: "behaviour")

    def gaiaquery(self):
        self.append(lambda x: "gaiaquery")

    def render(self, registry: VariableRegistry):
        return " ".join(map(lambda e: e(registry), self))